hazelcast/src/main/java/com/hazelcast/client/impl/protocol/ClientMessageReader.java

```diff
@@ -93,4 +119,29 @@ private boolean accumulate(ByteBuffer src, byte[] dest, int length) {
         return false;
     }
 
+    public static Builder builder() {
```

Comment:
why did we need to introduce a new class builder? I could not see the specific need.

Reply:
It was for extensibility and clearness reasons. The builder avoids confusion when properties have the same type - users can then simply screw their order in the constructor.

Nevertheless, as only one property remains now, I've changed the implementation to provide it in the constructor.